README.md

```diff
@@ -52,8 +52,12 @@ Chat Room
 
 Want to chat with other members of the PowerShell community?
 
+We have a Gitter Room which you can join below
```

Comment:
GitHub renders the badge to the right on my machine, so below is confusing.

I suggest: s/join below/join:/